de.bund.bfr.knime.fsklab.deprecatednodes/src-1_7_2/de/bund/bfr/knime/fsklab/nodes/v1_7_2/joiner/JoinerNodeModel.java

```diff
@@ -336,15 +347,27 @@ private void creatRelationList(String relation, JoinerViewValue joinerProxyValue
           jR.setLanguage_written_in(sourceTargetRelation.getString("language_written_in"));
         }
         if (sourceTargetRelation.containsKey("sourceParam")) {
-          jR.setSourceParam(getObjectFromJson(sourceTargetRelation.get("sourceParam").toString(),
-              Parameter.class));
+        	try {
+        		jR.setSourceParam(getObjectFromJson(sourceTargetRelation.get("sourceParam").toString(),
```

Comment:
We could use instead the convenience method getString of JsonObject https://docs.oracle.com/javaee/7/api/javax/json/JsonObject.html#getString-java.lang.String-